config/routes.rb

```diff
@@ -1,11 +1,6 @@
 Rails.application.routes.draw do
-  # get 'posts/index'
-  # get 'posts/show'
-  # get 'users/index'
-  # get 'users/show'
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root "users#index"
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-  end
```

Comment:
- [ ] According to the previous reviewer, you were supposed to add these pair of routes which you did after the review but creating a new branch might have removed it. Please kindly add the routes back.